### ISSUE TITLE
please release a version compatible with fgl 5.6.0.0

### ISSUE THREAD

**Issue by clinty** [ivan-m/graphviz#26]

**Comment by ivan-m** [comment]
Sorry, I've been on holidays (with almost no internet) and I'm trying to resolve some parsing issues that arose from #20 first.  Should be out in the next few days.

**Comment by ilovezfs** [comment]
Related https://github.com/haskell-infra/hackage-trustees/issues/114

**Comment by ilovezfs** [comment]
@ivan-m gentle ping on this.

**Comment by clinty** [comment]
Any news?

**Comment by ivan-m** [comment]
Sorry, life got in the way. Maybe this weekend.

**Comment by clinty** [comment]
Looks like hspec branch is merged and working?

**Comment by ivan-m** [comment]
Yes, but I still need to implement the test cases for the bugs I'm trying to fix (which was blocked by my old test-suite being difficult to update, hence the change to hspec).

**Comment by clinty** [comment]
Yay!
